examples/compose/plot_transformed_target.py

```diff
@@ -159,49 +160,51 @@
 ax1.set_xlabel('Target')
 ax1.set_title('Transformed target distribution')
 
-f.suptitle("Boston housing data: distance to employment centers", y=0.035)
+f.suptitle("Ames housing data: selling price", y=0.035)
 f.tight_layout(rect=[0.05, 0.05, 0.95, 0.95])
 
 X_train, X_test, y_train, y_test = train_test_split(X, y, random_state=1)
 
 ###############################################################################
 # The effect of the transformer is weaker than on the synthetic data. However,
-# the transform induces a decrease of the MAE.
+# the transform induces an increase in R^2 and large decrease of the MAE.
 
 f, (ax0, ax1) = plt.subplots(1, 2, sharey=True)
 
 regr = RidgeCV()
 regr.fit(X_train, y_train)
 y_pred = regr.predict(X_test)
 
-ax0.scatter(y_test, y_pred)
-ax0.plot([0, 10], [0, 10], '--k')
+ax0.scatter(y_test, y_pred, s=8)
+ax0.plot([0, 7e5], [0, 7e5], '--k')
 ax0.set_ylabel('Target predicted')
 ax0.set_xlabel('True Target')
-ax0.set_title('Ridge regression \n without target transformation')
-ax0.text(1, 9, r'$R^2$=%.2f, MAE=%.2f' % (
+ax0.set_title('Ridge regression \n without target transformation', pad=18)
```

Comment:
The `pad` keyword argument is causing the doc build to fail with older, yet supported versions of matplotlib.

Reply:
Yes, see my comment: https://github.com/scikit-learn/scikit-learn/pull/16741#issuecomment-602112331

I don't understand why people can't seem to see my comments on PRs - this is the second time this has happened! Do you think I changed some setting accidentally?